### DISCUSSION TITLE
[REQUEST] Freescout

### DISCUSSION THREAD

**Discussion by NorthantsAndy** [tteck/Proxmox#2163] [Ideas]
[FreesScout.net](https://freescout.net/)

FreeScout is the super lightweight and powerful [free open source help desk](https://freescout.net/blog/open-source-helpdesk-system/) and shared inbox written in PHP (Laravel framework). It is a self hosted clone of HelpScout.

**Comment by OddSquirrel** [comment]
> FreeScout is the super lightweight and powerful [free open source help desk](https://freescout.net/blog/open-source-helpdesk-system/)

It's also the only one that has the nerve to post a "comparison" in their very own blog where, oh wonder, they incidentally hit 1st place. 😇

Seriously, I was almost wowed by the design, and then I saw this. Couldn't get more fishy. Looks like some random team from India trying to cash in on Dark Mode and countless other shenanigans. Oh well, to each their own. 😋
